pkg/server/plugin/nodeattestor/x509pop/x509pop_test.go

```diff
@@ -427,3 +442,12 @@ func unmarshal(t *testing.T, data []byte, obj any) {
 func expectNoChallenge(context.Context, []byte) ([]byte, error) {
 	return nil, errors.New("challenge is not expected")
 }
+
+func sortX509PopSelectors(s []*common.Selector) {
```

Comment:
I think you can use https://github.com/spiffe/spire/blob/1b343d9c4bbd32178c83aeee412dd22916a84fd6/pkg/common/util/sort.go#L45 for this.